### DISCUSSION TITLE
Is Popover componenet a good choice for tooltips ?

### DISCUSSION THREAD

**Discussion by ahmedosama5200** [tailwindlabs/headlessui#788] [Help]
Docs say "Popovers are perfect for floating panels with arbitrary content like navigation menus, mobile menus and flyout menus."

Can I just use them for tooltips ?

**Comment by jonathanstanley** [comment]
so far, appears not. see https://github.com/tailwindlabs/headlessui/discussions/436

**Comment by duongital** [comment]
I think Popover shouldn't go for Tooltip, the behavior of Tooltip is quite different. For example we need it move out of root and display in portal section to avoid overlap elements.
